package.json

```diff
@@ -30,7 +30,10 @@
     "url": "https://github.com/jstransformers/jstransformer-lodash/blob/master/LICENSE.md"
   },
   "dependencies": {
-    "lodash": "^3.6.0"
+    "lodash.assign": "^3.2.0",
```

Comment:
Since you're using `^` you can just make all the versions `^3.0.0` for simplicity.